test/features/deploy_service_resources.feature

```diff
@@ -0,0 +1,40 @@
+@resoures
+Feature: Deploy the service by configuring the resource requests and limits
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  @runtimerequests @runtimelimits
```

Comment:
can you put each tag on a new line for better visibility ?

Reply:
done